Replace env with params and add additional

jcpowermac opened the pull request:
- Added env vars for dev and stage project name
- Added Dev OpenShift uri
- Added Checkout path restriction

aweiteka commented on Jenkinsfile:
```diff
@@ -6,10 +6,24 @@ pipeline {
+        DEV_PROJECT = "lifecycle"
+        STAGE_PROJECT = "lifecycle"
         STAGE_URI = "insecure://openshift-ait.e2e.bos.redhat.com:8443"
+        DEV_URI = "insecure://osemaster.sbu.lab.eng.bos.redhat.com:8443"
```
Should we put all of these into env.VAR that is passed in from the OpenShift BC?

aweiteka replied:
Or maybe these will be overridden by those in the BC

aweiteka replied:
AFICT, openshift BC env vars are mapped as both params.KEY and regular environment vars.

jcpowermac replied:
@aweiteka let me test this

aweiteka commented on Jenkinsfile:
```diff
@@ -6,10 +6,24 @@ pipeline {
     stages {
+        stage('Checkout') {
+            steps {
+				checkout([$class: 'GitSCM',
```
nit: indent

jcpowermac committed:
Add additional Jenkinsfile parameters

- Replace env with params
- Added params for dev and stage project name
- Added Checkout path restriction
- Secret string is a param

jcpowermac committed:
comment out params, will be used for testing
